attributes/default.rb

```diff
@@ -1,5 +1,11 @@
 
 # Override cookbook attributes to install Java 7
-override['java']['openjdk_packages'] = [
-  "openjdk-7-jdk", "openjdk-7-jre-headless"
-]
+#override['java']['openjdk_packages'] = [
+#  "openjdk-7-jdk", "openjdk-7-jre-headless"
+#]
```

Comment:
Can you please uncomment this? This is needed to install Java 7; I was having trouble actually getting the proper version to install, so I had to settle with overriding attributes in the `java` recipe.